src/_posts/databases/postgresql/timescaledb/2000-01-01-dump-restore.md

```diff
@@ -0,0 +1,41 @@
+---
+title: TimescaleDB
+nav: Dump and restore
+modified_at: 2022-02-18 00:00:00
+tags: timescale databases postgresql extensions dump restore
+index: 2
+---
+
+{% warning %}
+  The backup restoration process may put the base in an undesirable state.
+  Pay attention to the restoration process.
+{% endwarning %}
+
+## Dump
+
+The only dump available of a PostgreSQL using the TimescaleDB extension are the
+[on demand backups]({% post_url databases/postgresql/2000-01-01-start %}#on-demand-backups).
+The reason is that the restoration process must be done by the Scalingo support using
+these specific backups, see below.
+
+Note that [PiTR]({% post_url databases/postgresql/2000-01-01-start %}#point-in-time-recovery)
+backups are still working normally.
+
+## Restore
+
+The backup restoration on TimescaleDB is requiring a preparation of the PostgreSQL database
+which needs admin rights. That's why you have to ask our support to handle the process.
+Please, specify in your support request the timestamp of the backup.
+
+## Warnings
+
+Several points are important to take into account.
+
+It may happen some trouble due to a PostgreSQL and TimescaleDB version mismatch
+between the time the backup was made and the database you are trying to restore to.
```

Comment:
This sentence is unclear and should be rephrased

Reply:
I'm sorry but I really don't understand this sentence. Can you explain in French what you want to explain here?

Reply:
It is clearer to refer to the doc with all the details.